waggle-dance/src/main/service/waggle-dance.conf

```diff
@@ -1,4 +1,4 @@
 #Spring boot service conf: see http://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html#deployment-script-customization-conf-file
 LOG_FOLDER=/var/log/waggle-dance/
-JAVA_OPTS="-Xmx12g -XX:+UseG1GC -XX:G1ReservePercent=15 -Dlog4j.configurationFile=/opt/waggle-dance/conf/log4j2.xml -Dlogging.config=/opt/waggle-dance/conf/log4j2.xml"
+JAVA_OPTS="-Xmx12g -XX:+UseG1GC -XX:G1ReservePercent=15 -Dloader.path=lib,/opt/waggle-dance/lib/ -Dlog4j.configurationFile=/opt/waggle-dance/conf/log4j2.xml -Dlogging.config=/opt/waggle-dance/conf/log4j2.xml"
```

Comment:
does this not work with the jar layout?

Reply:
Not with how we have it prior to this PR